spec/requests/posts_spec.rb

```diff
@@ -3,7 +3,9 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
```

Comment:
- [ ] The screenshot below, from your project's requirements outlined clearly the different features we are supposed to test for the `User post index page` . Keenly visualized them line after another and write unit tests that affirm all the scenarios:



![Screenshot from 2022-09-06 02-12-06](https://user-images.githubusercontent.com/63087491/188526954-71400c06-4c95-4647-8387-8f65eea9f9a7.png)

Reply:
same thing here